Paco/src/com/pacoapp/paco/js/bridge/JsUtil.java

```diff
@@ -72,15 +76,18 @@ public static SQLQuery convertJSONToPOJO(String queryJson) throws JSONException
           }
         }
       }
-
+      
+      //Only if we have sort order, we should accept limit value. This is because we cannot limit 
+      //records using the sql lite query builder query method. What we can do is hack the order by clause to have the string
+      //limit 'x'. We can limit records only when we have sortOrder column.
       if (criteriaQueryObj.has("order")) {
         sortOrder = criteriaQueryObj.getString("order");
+        if (criteriaQueryObj.has("limit")) {
```

Comment:
The query builder's query method does not take in a parameter for 'limit'. So what we do is append the sortOrder clause with "limit" value. With this hack, when we do not have sortOrder, we cannot  append the limit clause. It results in query.....'select ...from table sort by limit 100'  with compilation issue. I will add this clause to the documentation.